### DISCUSSION TITLE
adding customize font size and width option.

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#890] [Ideas]
Hey, @fuma-nama are you working on any thing right now ? No just had this idea of like adding a setting option in the docs just a dropdown below the footer of docs sidebar like the one wikipedia has. Just to change the size and width of font. 

![Screenshot 2024-09-22 221332](https://github.com/user-attachments/assets/8983013f-76bd-472b-854d-0dbcf6308734)

and it will look like this 

![Screenshot 2024-09-22 222910](https://github.com/user-attachments/assets/54408248-2413-4e8d-8893-61b8b9d918c8)

It seems like a good option to change the font size and it look a little bit small sometimes. 

I don't know if you would add it in the main fumadocs as optional add in but I want to add that, is there a way that from my `docs > page.tsx` file that I can pass the props in the configuration to modify it. Or should I just create a component for it and import it ? 

How should I do this task  ?

**Comment by fuma-nama** [comment]
I think you should just modify it with CSS
